docs/UserGuide.md

```diff
@@ -9,6 +9,52 @@ NetworkBook is a **desktop contact book application**. You can use it to network
 
 ### <u>Category 1 - Add contact information</u>
 
+### Create new contact: `create /name [name] [/optional fields]`
```

Comment:
For consistency with other headers, could u remove the "[name]" so that the header only contains handles but not field values